### PR TITLE
Fix remaining marketing links

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: lms/templates/register-form.html [hunk 1]
```diff
@@ -247,7 +247,7 @@ <h2 class="sr">${_("Account Acknowledgements")}</h2>
         <label for="tos-yes">${_('I agree to the {link_start1}Terms of Service{link_end} and {link_start2}Privacy Policy{link_end}').format(
           link_start1='<a href="{url}" class="new-vp" tabindex="-1">'.format(url=marketing_link('TOS')),
           link_end='</a>',
-          link_start2='<a href="{url}#privacy" class="new-vp" tabindex="-1">'.format(url=marketing_link('TOS')),
+          link_start2='<a href="{url}" class="new-vp" tabindex="-1">'.format(url=marketing_link('PRIVACY')),
```
Comment (stvstnfrd) [inline lms/templates/register-form.html]: We're not using this file anymore, but I'm updating for completeness.

**Comment by stvstnfrd** [comment]
@caesar2164 Please review.

**Comment by caesar2164** [comment]
nice catch, 👍

**Comment by caesar2164** [comment]
The tests failed because of the usual `gitlog_pagination` error, disregard and merge at your leisure.

**Comment by stvstnfrd** [comment]
retest this please

**Pull request merged**